app/views/host_aggregate/remove_host_select.html.haml

```diff
@@ -0,0 +1,27 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Remove Host from Host Aggregate')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Host')
+      .col-md-8
+        = select_tag("host_id",
+          options_for_select([["<#{_('Choose')}>", nil]] + @host_choices.sort),
+                      "ng-model"                    => "hostAggregateModel.host_id",
+                      "required"                    => "",
+                      :miqrequired                  => true,
+                      :checkchange                  => true,
+                      "selectpicker-for-select-tag" => "")
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
+          = render :partial => "layouts/angular/x_custom_form_buttons_angular",
+                   :locals  => {:button_label           => "Remove",
```

Comment:
this needs to be `N_("Remove")` otherwise, that `_(button_label)` won't work.

Same for `Add`.

And please change `_("#{button_label}")` to `_(button_label)` :)